tutorials/2_sprite_animations_gestures/README.md

```diff
@@ -0,0 +1,229 @@
+
+So, to change that, we will now add some interactivity to our game and make the button tapable/clickable.
+
+Flame provides several input handlers, which you can check with more in depth on [our docs](https://github.com/flame-engine/flame/blob/main/doc/input.md). For this tutorial, we will be using the `TapDetector` which enables us to detect taps on the screen, as well as mouse click when running on web or desktop.
+
+All Flame input detectors are mixins which can be added to your game, enabling you to override listener methods related to that detector. For the `TapDetector`, we will need to override three methods:
+
+ - `onTapDown`: Called when touch/click has started, i.e., the user just touced the screen or clicked the mouse button.
+ - `onTapUp`: Called when the touch/click has stop occurring because the event was released, i.e., the user lifted the finger from the screen or released the mouse button.
+ - `onTapCancel`: Called when the event was cancelled. This can happen for several reasons; one of the most common is when the event has changed into another type, for example the user started to move the finger/mouse and the touch event now turned into a pan/drag. Usually, we can just treat this event as being the same as `onTapUp`.
+
+Now that we have a better understanding of `TapDetector` and the events that we will need to handle, let's implement it on the game:
+
+```dart
+// We need to add our `TapDetector` mixin here
+class MyGame extends Game with TapDetector {
+  // Variables declaration, onLoad and render methods omited...
+
+  @override
+  void onTapDown(TapDownDetails details) {
+    // On tap down we need to check if the event ocurred on the
+    // button area. There are several ways of doing it, for this
+    // tutorial we do that by transforming ours position and size
+    // vectors into a dart:ui Rect by using the `&` operator, and
+    // with that rect we can use its `contains` method which checks
+    // if a point (Offset) is inside that rect
+    final buttonArea = buttonPosition & buttonSize;
+
+    isPressed = buttonArea.contains(details.localPosition);
+  }
+
+  // On both tap up and tap cancel we just set the isPressed
+  // variable to false
+  @override
+  void onTapUp(TapUpDetails details) {
+    isPressed = false;
+  }
+
+  @override
+  void onTapCancel() {
+    isPressed = false;
+  }
+
+  // Finally, we just modify our update method so the animation is
+  // updated only if the button is pressed
+  @override
+  void update(double dt) {
+    if (isPressed) {
+      runningRobot.update(dt);
+    }
+  }
+}
+```
+
+If we run our game again, we should have the full example running, with our on/off button for our little vampire robot.
+
+And with that, we finished this tutorial. Now, with an understanding of sprites, animations and gestures, we can start on building more interactive and beautiful games.
```

Comment:
I'm thinking we could have small quick hooks into the next tutorial on the last paragraphs. but we can do that later.